LanguageApp/app/screens/CategoriesScreen.js

```diff
@@ -1,35 +1,41 @@
 import React, { useState } from "react";
-import { FlatList, View, StyleSheet } from "react-native";
+import { FlatList, View } from "react-native";
 
-import AppTitle from "../components/AppTitle";
 import ListItem from "../components/ListItem";
 import ListItemSeparator from "../components/ListItemSeparator";
-import Screen from "../components/Screen";
+import routes from "../navigation/routes";
 
 const initialCategories = [
-  "Time",
-  "Weather",
-  "Food",
-  "Animals",
-  "Activities",
-  "Travel",
+  { name: "Time", icon: "clock" },
+  { name: "Weather", icon: "weather-partly-cloudy" },
+  { name: "Animals", icon: "dog-side" },
+  { name: "Travel", icon: "wallet-travel" },
+  { name: "Food", icon: "food" },
+  { name: "Activities", icon: "lightbulb-on-outline" },
```

Comment:
All of this will be pulled from Firebase?  Or will this remain hardcoded since we're manually inputting scenarios?